vendor/k8s.io/kube-openapi/pkg/handler/handler.go

```diff
@@ -259,3 +282,16 @@ func BuildAndRegisterOpenAPIVersionedService(servePath string, webServices []*re
 	}
 	return o, o.RegisterOpenAPIVersionedService(servePath, handler)
 }
+
+func (c *cache) Get() ([]byte, string, error) {
+	c.once.Do(func() {
+		bytes, etag, err := c.BuildCache()
+		c.bytes = bytes
+		c.etag = etag
+		c.err = err
```

Comment:
You can turn that into a single line:
```suggestion
		c.bytes, c.etag, c.err = c.BuildCache()
```

Reply:
Good point - fixed!